Source/monster.cpp

```diff
@@ -671,7 +670,7 @@ void PlaceMonster(int i, int mtype, int x, int y)
 	}
 	dMonster[x][y] = i + 1;
 
-	rd = random_(90, 8);
+	direction rd = (direction)random_(90, 8);

```

Comment:
```suggestion
	direction rd = static_cast<direction>(random_(90, 8));

```

```suggestion
		Monst->_mdir = static_cast<direction>(random_(89, 8));

```

Here we could also use NUM_Direction.